### PR TITLE
added * for mandatory fields

### PR DESCRIPTION
**Pull request by Hitendra10**

### CODE REVIEW

File: app/controllers/signup.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default class SignupController extends Controller {
   @tracked fields = [
     {
       id: 'first_name',
-      label: 'First Name',
+      label: 'First Name. *',
```
[the remark this answers is not in the document]
Comment (Hitendra10) [reply app/controllers/signup.js:46]: should i remove the dots?

**Review by swarajpure — CHANGES_REQUESTED** [review]
There is an open comment on this PR
@Hitendra10 are you still working on this?

**Comment by Hitendra10** [comment]
No, i am no longer working on this.

**Comment by rohan09-raj** [comment]
Closing the Pull Request as the author is no longer working on the PR
https://github.com/Real-Dev-Squad/website-my/pull/137#issuecomment-1051857396

**Pull request closed without merging**